build/SynologyNAS/toolkit/source/megacmdpkg/SynoBuildConf/install

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash 
+# Copyright (C) 2000-2016 Synology Inc. All rights reserved.
```

Comment:
synology inc?

Reply:
removed, thanks  -that was left over from the sample project